src/module_2/eda_groceries_2.md

```diff
@@ -0,0 +1,186 @@
+**EDA GROCERIES 2**
+
+Imports
+
+
+```python
+import boto3
+import os
+from dotenv import load_dotenv
+from botocore.exceptions import ClientError
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import plotly.express as px
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
+```
+
+Downloading data
+
+
+```python
+# load_dotenv()
+
+# def download_file_from_s3(bucket_name, file_key, download_path):
+#     # Create a session using your AWS credentials
+#     session = boto3.Session(
+#         aws_access_key_id=os.getenv('AWS_ACCESS_KEY_ID'),
+#         aws_secret_access_key=os.getenv('AWS_SECRET_ACCESS_KEY'),
+#         region_name=os.getenv('AWS_REGION')
+#     )
+
+#     # Create an S3 client
+#     s3 = session.client('s3')
+
+#     try:
+#         # Download the file
+#         s3.download_file(bucket_name, file_key, download_path)
+#         print(f"Downloaded {file_key} from {bucket_name} to {download_path}")
+#     except Exception as e:
+#         print(f"Error downloading file: {e}")
+
+
+# bucket_name = 'zrive-ds-data'
+# file_key = 'groceries/box_builder_dataset/feature_frame.csv'
+# download_path = 'feature_frame.csv'
+
+
+# download_file_from_s3(bucket_name, file_key, download_path)
```

Comment:
Same as before. If we don't want it in our notebook, maybe we should create a different script to execute it outside